doc_components/styles.ts

```diff
@@ -97,12 +97,11 @@ const styles = {
     `block pl-2 py-0.5 lg:(table-cell w-48 pl-4 pr-2 py-1.5)`,
   moduleIndexPanelCell: apply`block lg:table-cell ${panelStyles}`,
   modulePathIndex: apply`rounded-lg w-full border border-secondary`,
-  modulePathIndexHeader: apply`flex justify-between items-center`,
+  modulePathIndexHeader: apply`flex justify-between items-center py-3.5 pr-5`,
+  SymbolDocHeader: apply`flex justify-between items-center py-3.5`,
```

Comment:
We should be consistent in casing:

```suggestion
  symbolDocHeader: apply`flex justify-between items-center py-3.5`,
```

Also, they are all alphabetically ordered.